docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: "3.6"
+services:
+  stacks-blockchain:
+    image: blockstack/stacks-blockchain:2.0.11.1.0
+    ports:
+      - "20443:20443"
+      - "20444:20444"
+    volumes:
+      - ./stacks-node/persistent-data/stacks-blockchain:/root/stacks-node/data
```

Comment:
I think this is not working properly, so if the server is restarted we need to start again at 0.
To fix before merging